pdfarranger/pdfarranger.py

```diff
@@ -27,6 +27,8 @@
 import gc
 import subprocess
 from urllib.request import url2pathname
+if os.name == 'posix':
+    from ctypes import CDLL as libloader
```

Comment:
I would just import ctypes here (i.e not test os.name and not rename CDLL to something else)